spec/models/manageiq/providers/ibm_cloud/vpc/cloud_manager/cloud_database.spec

```diff
@@ -18,6 +18,20 @@ describe ManageIQ::Providers::IbmCloud::VPC::CloudManager::CloudDatabase do
 
     before { allow(ems).to receive(:with_provider_connection).and_yield(connection) }
 
+    context '#create_cloud_database' do
+      it 'creates the cloud database' do
+        allow(connection).to receive_message_chain(:resource, :manager, :collection, :find).and_return({:id => "test123"})
+        allow(connection).to receive_message_chain(:resource, :controller).and_return(resource_controller)
```

Comment:
Since this is done by both specs you could move this up to the common `before` block:
```ruby
    before do
      allow(ems).to receive(:with_provider_connection).and_yield(connection)
      allow(connection).to receive_message_chain(:resource, :controller).and_return(resource_controller)
    end
```